.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,112 @@
+name: Continuous integration
+on: [push]
+env:
+  TOX_PARALLEL_NO_SPINNER: 1
+  PYTEST_ADDOPTS: -x
+jobs:
+  backend:
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:11.5-alpine
+        ports:
+        - 5432:5432
+      elasticsearch:
+        image: hypothesis/elasticsearch:latest
+        ports:
+        - 9200:9200
+        env:
+          discovery.type: single-node
+    steps:
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.6.9
+    - name: Update pip
+      run: python -m pip install --upgrade pip
+    - name: Install tox
+      run: python -m pip install tox
+    - name: Checkout git repo
+      uses: actions/checkout@v2
+    - name: Create test databases
+      run: psql -U postgres -h localhost -p 5432 -c 'CREATE DATABASE htest'
+    - name: Cache the .tox dir
+      uses: actions/cache@v2
+      with:
+        path: |
+          .tox/checkformatting
+          .tox/tests
+          .tox/coverage
+        key: ${{ runner.os }}-tox-backend-${{ hashFiles('tox.ini') }}-${{ hashFiles('requirements*') }}-${{ hashFiles('setup.py') }}-${{ hashFiles('setup.cfg') }}
```

Comment:
The `hashFiles` function does accept multiple path pattern arguments [according to the docs](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#hashfiles) so this line could perhaps be simplified to `${{ runner.os }}-tox-backend-$${{ hashFiles('tox.ini', 'requirements*', ...) }}`.

Were you trying to create a hierarchy of cache keys though? If so I'm not sure about the value of having this many levels.

Reply:
Not a hierarchy, I was just trying to cover all the files that could possibly contribute to a change in requirements across any of our projects so that I can just copy-paste the same key for all projects. Rather have false-positives and unnecessarily re-sync requirements, than have false negatives

Reply:
Fixed - I had thought that `hashFiles()` could only take one argument